design-proposals/dedicated-migration-network.md

```diff
@@ -0,0 +1,94 @@
+# Overview
```

Comment:
It may be useful to mention for the less knowledgeable reader that the migration is conducted at the IP level between the virt-handlers and not directly from the virt-launchers. It is not that trivial to understand this part and readers may wrongly assume the secondary network is on the virt-launcher and not the virt-handler.